test/cypress/tests/ui/Placeholders.cy.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * Text will be passed as a placeholder to the editor
+ */
+const PLACEHOLDER_TEXT = 'Write something or press / to select a tool';
+
+describe('Placeholders', function () {
+  it('should be shown near first block if passed via editor config', function () {
+    cy.createEditor({
+      placeholder: PLACEHOLDER_TEXT,
+    });
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .getPseudoElementContent('::before')
+      .should('eq', PLACEHOLDER_TEXT);
+  });
+
+  it('should be shown when editor is autofocusable', function () {
+    cy.createEditor({
+      placeholder: PLACEHOLDER_TEXT,
+      autofocus: true,
+    });
+
+    cy.get('[data-cy=editorjs]')
+      .find('.ce-paragraph')
+      .getPseudoElementContent('::before')
+      .should('eq', PLACEHOLDER_TEXT);
+  });
```

Comment:
So in both cases when I pass a placeholder, it will be shown. It doesn't really depend on the autofocus param. The second case is redundant then.

Reply:
in previous version `autofocus` was leading placeholder to disappear, so I've added this test.